ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParsePayloadTest.java

```diff
@@ -322,4 +322,26 @@ public void testPingSplitting() throws IOException {
 
     pipeline.run();
   }
+
+  @Test
+  public void testMessagesDropped() throws IOException {
+    // ParsePayload should not output anything to output or error when
+    // MessageShouldBeDroppedException is thrown
+    String schemasLocation = TestConstant.SCHEMAS_LOCATION;
+    SchemaStoreSingletonFactory.clearSingletonsForTests();
+
+    final List<PubsubMessage> input = Arrays.asList(
+        new PubsubMessage(Json.asBytes(ImmutableMap.of("payload", ImmutableMap.of())),
+            ImmutableMap.of("document_namespace", "drop", "document_type", "account-ecosystem")),
+        new PubsubMessage(Json.asBytes(ImmutableMap.of("payload", ImmutableMap.of())), ImmutableMap
+            .of("document_namespace", "firefox-desktop", "document_type", "background-update")));
```

Comment:
```suggestion
    final List<PubsubMessage> input = Arrays
        .asList(
            new PubsubMessage(Json.asBytes(ImmutableMap.of("payload", ImmutableMap.of())),
                ImmutableMap.of("document_namespace", "drop", "document_type",
                    "account-ecosystem")),
            new PubsubMessage(Json.asBytes(ImmutableMap.of("payload", ImmutableMap.of())),
                ImmutableMap.of("document_namespace", "firefox-desktop", "document_type",
                    "background-update")),
            new PubsubMessage(
                Json.asBytes(ImmutableMap.of("payload", ImmutableMap.of(), "search_query",
                    "somequery", "matched_keywords", "query")),
                ImmutableMap.of("document_namespace", "contextual-services", "document_type",
                    "quicksuggest-impression")));
```
Optional, should make codecov happy.

Reply:
codecov still considers this a partial coverage.  Maybe it needs to run each combination of the OR?  It doesn't really matter though